src/foam/parse/QueryParserUserTest.js

```diff
@@ -0,0 +1,62 @@
+foam.CLASS({

```

Comment:
It's okay that you modelled it, but you don't need to. You could just create a Java class which extends Test and overrides the runTest method.